### PR TITLE
Building tabbed interface

### PR DESCRIPTION
**Pull request by kcalva**

### CODE REVIEW

File: app/controllers/users_controller.rb
Comment (brvarner) [inline app/controllers/users_controller.rb:1]: very elegant DRY solution!

File: app/models/user.rb
Comment (brvarner) [inline app/models/user.rb:1]: great!

File: app/views/photos/_photo.html.erb
Comment (brvarner) [inline app/views/photos/_photo.html.erb:1]: Photo card looks really good!

File: app/views/shared/_tab_interface.html.erb
Comment (brvarner) [inline app/views/shared/_tab_interface.html.erb:1]: great work making this a partial!

File: app/views/users/_users.html.erb
Comment (brvarner) [inline app/views/users/_users.html.erb:1]: Nice and concise!

File: config/routes.rb
Comment (brvarner) [inline config/routes.rb:1]: Great order on these!

**Review by brvarner — APPROVED** [review]
Looks good!

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,5 +1,24 @@
 class UsersController < ApplicationController
+  before_action :set_user
```
Comment (CarlosCastanedaDev) [inline app/controllers/users_controller.rb:2]: Great job creating a before_action and keeping your code DRY

**Review by CarlosCastanedaDev — APPROVED** [review]
Looks great! Nice job keeping your code DRY

**Pull request merged**